rollrus_test.go

```diff
@@ -154,3 +155,12 @@ func TestTriggerLevels(t *testing.T) {
 		t.Fatal("Expected Levels() to return newLevels")
 	}
 }
+
+func TestWithIgnoredErrors(t *testing.T) {
+	hook := NewHook("foobar", "testing", WithIgnoredErrors(io.EOF))
+
+	want := map[error]bool{io.EOF: true}
+	if !reflect.DeepEqual(hook.ignoredErrors, want) {
```

Comment:
Is this test useful? It's basically testing that you set some internal variable vs. testing to see if logging with the hook skips io.EOF errors. Also a negative test would also be ++